augur/utils.py

```diff
@@ -713,7 +703,7 @@ def read_strains(*files, comment_char="#"):
     """
     strains = set()
     for input_file in files:
-        with open(input_file, 'r', encoding='utf-8') as ifile:
+        with open_file(input_file, 'r', encoding='utf-8') as ifile:
```

Comment:
We have to drop the `encoding` keyword argument here because `xopen` does not support this. This is more evidence in support of implementing our own compression backend.